packages/core/src/utilities/VoxelManager.ts

```diff
@@ -468,6 +476,7 @@ export default class VoxelManager<T> {
    */
   public static addBounds(bounds: BoundsIJK, point: Point3) {
     if (!bounds) {
+      console.debug('🚀 ~ reset bounds');
```

Comment:
If you are going to leave console statements around, could you introduce a logging library that can be redirected/replaced?  I'd love to use log.debug(....) and have log be a type specific to VoxelManager so we can actually figure out what is going on and turn on specific logging bits at runtime.

Reply:
It was a mistake, removed it, but logging lib is needed